app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ReviewsController < ApplicationController
+  before_action :find_card, only: [:trainer]
+
+  def new
+    @card = Card.reminder
+  end
+
+  def trainer
+    if right_translation?
+      flash[:success] = t('messages.home.success')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.